src/provideInlineCompletionItems.ts

```diff
@@ -24,7 +25,8 @@ export default async function provideInlineCompletionItems(
     if (
       !completionIsAllowed(document, position) ||
       !isValidMidlinePosition(document, position) ||
-      !getShouldComplete()
+      !getShouldComplete() ||
+      shouldBlockCompletions()
```

Comment:
r u sure?

Reply:
wdym? this is supposed to block completions no? if `shouldBlockCompletions` then yes. if you are saying that this isn't the semantics, then explain

Reply:
i think Daniel asked  not to block

Reply:
it is handled in the function